xExtension-ReadingTime/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Lapineige
```

Comment:
;-)

Reply:
? (I don't understand ^^)

Reply:
Well, it seems that I started this a while ago ^^
(end of 2016)

Reply:
I often write something like `2016 - 2018` in such a situation